Assets/Scripts/Enemies/Sentry.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Sentry : Enemy
+{
+    public GameObject bulletPrefab;
+    private GameObject player;
+    private Transform gun;
+    private float bulletSpeed = 0.25f;
+
+    void Start()
+    {
+        gun = transform.GetChild(1).transform;
```

Comment:
Again I would recommend either just setting this in the  inspector of the actual prefab itself rather than in the start in cas e the prefab gets changed, however is not a big of a deal since its not trying to get a component that might be missing, it can still end in a out of bounds exception cuz of the hardcoded 1